src/vrv.cpp

```diff
@@ -180,10 +180,8 @@ LogLevel StrToLogLevel(const std::string &level)
 
 bool LogBufferContains(const std::string &s)
 {
-    std::vector<std::string>::iterator iter = logBuffer.begin();
-    while (iter != logBuffer.end()) {
-        if ((*iter) == s) return true;
-        ++iter;
+    for (const std::string &logStr : logBuffer) {
+        if (logStr == s) return true;
```

Comment:
Looks much cleaner. Even nicer would be to use `std::any_of` here. But maybe that's something for another refactoring chapter.

Reply:
Yes, I'll leave it like this.